### PR TITLE
[KUBECTL-808] kubectl logs command with label selector option doesn't…

### PR DESCRIPTION
**Pull request by bseenu**
… print error messages

**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes # https://github.com/kubernetes/kubectl/issues/808

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```
None
```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go [hunk 1]
```diff
@@ -275,6 +275,9 @@ func (o *LogsOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []str
 			return errors.New("expected a resource")
 		}
 		o.Object = infos[0].Object
+		if o.Selector != "" && len(o.Object.(*corev1.PodList).Items) == 0 {
+			return errors.New(fmt.Sprintf("no resources found in %s namespace", o.Namespace))
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go:279]: `fmt.Errorf`

**Review by soltysh — CHANGES_REQUESTED** [review]
Sync with @cprayer about https://github.com/kubernetes/kubernetes/pull/89688

File: staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go (hunk 1 above)
Comment (bseenu) [reply staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go]: 👍

**Pull request closed without merging**